crates/telio-traversal/src/endpoint_providers/upnp.rs

```diff
@@ -545,9 +553,11 @@ impl<Wg: WireGuard, I: UpnpEpCommands, E: Backoff> State<Wg, I, E> {
     }
 
     fn create_random_endpoint_ports(&mut self) {
-        self.wg_port_mapping.external = rand::thread_rng().gen_range(1024..49151);
+        self.wg_port_mapping.external =
+            rand::thread_rng().gen_range(MIN_EPHEMERAL_PORT_RANGE..MAX_EPHEMERAL_PORT_RANGE);
```

Comment:
Hmm, maybe this could be just a const range?

Reply:
Yes it could, however I don't think there is any benefit, if not for worse in case of readability/flexibility in accessing each value individually..

Reply:
Is `MIN_EPHEMERAL_PORT_RANGE..MAX_EPHEMERAL_PORT_RANGE` more readable than simply `EPHEMERAL_PORT_RANGE`? I don't think so, but this is a nitpick anyway.